lib/foreman_remote_execution/engine.rb

```diff
@@ -45,6 +45,8 @@ class Engine < ::Rails::Engine
                                             :'api/v2/job_templates' => [:update],
                                             :'api/v2/template_inputs' => [:create, :update, :destroy],
                                             :'api/v2/foreign_input_sets' => [:create, :update, :destroy]}, :resource_type => 'JobTemplate'
+          permission :edit_remote_execution_features, { :remote_execution_features => [:index, :show, :update],
```

Comment:
ok fine, it's just inconsistent with everything else we have (edit never grants view permission), I'll file it and we'll see if someone finds it confusing